v2_papers/2010/lim_internet.json

```diff
@@ -0,0 +1,285 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet Traffic Classification Demystified: On the Sources of the Discriminative Power",
+    "authors": [
+      "Lim, Yeon-sup",
+      "Kim, Hyun-chul",
+      "Jeong, Jiwoong",
+      "Kim, Chong-kwon",
+      "Kwon, Ted “Taekyoung\"",
```

Comment:
Not completely sure, but the error on the verifier might be because of these weird quotation marks.

Reply:
I just checked myself, the quotation marks are not the problem the verifier has. However, I still think they should just be removed.